### DISCUSSION TITLE
Copy/Paste not working: `y` `p`

### DISCUSSION THREAD

**Discussion by curkan** [nvim-tree/nvim-tree.lua#1867] [Q&A]
### Description

I try to copy the file through 'y', everything is fine, it says that it is added to the clipboard, then I press 'p' anywhere, and nothing happens. With what can be connected?
How do you know if a plugin is interfering?
Or is it something else

### Neovim version

```text
NVIM v0.8.1
Build type: Release
LuaJIT 2.1.0-beta3
```

### Operating system and version

macOS 13.1

### nvim-tree version

9e4c395

### Minimal config

```lua
vim.cmd [[set runtimepath=$VIMRUNTIME]]
vim.cmd [[set packpath=/tmp/nvt-min/site]]
local package_root = "/tmp/nvt-min/site/pack"
local install_path = package_root .. "/packer/start/packer.nvim"
local function load_plugins()
  require("packer").startup {
    {
      "wbthomason/packer.nvim",
      "nvim-tree/nvim-tree.lua",
      "nvim-tree/nvim-web-devicons",
      -- ADD PLUGINS THAT ARE _NECESSARY_ FOR REPRODUCING THE ISSUE
    },
    config = {
      package_root = package_root,
      compile_path = install_path .. "/plugin/packer_compiled.lua",
      display = { non_interactive = true },
    },
  }
end
if vim.fn.isdirectory(install_path) == 0 then
  print "Installing nvim-tree and dependencies."
  vim.fn.system { "git", "clone", "--depth=1", "https://github.com/wbthomason/packer.nvim", install_path }
end
load_plugins()
require("packer").sync()
vim.cmd [[autocmd User PackerComplete ++once echo "Ready!" | lua setup()]]
vim.opt.termguicolors = true
vim.opt.cursorline = true

-- MODIFY NVIM-TREE SETTINGS THAT ARE _NECESSARY_ FOR REPRODUCING THE ISSUE
_G.setup = function()
  require("nvim-tree").setup {}
end
```

### Steps to reproduce

1. nvim -nu /tmp/nvt-min.lua
2. :NvimTreeToggle
3. yank
4. try paste (no works for me)

**Comment by alex-courtis** [comment]
`y` and `Y` copy the names/paths - the actual text.

`c` copies the file itself.

I've gotten trapped by this myself a few times. Perhaps I should change those mappings... see `:help nvim-tree-mappings`

**Comment by omegahh** [comment]
I am wondering that how to paste file itself after pressing `c`

**Reply by alex-courtis** [reply]
`p` will paste the cut or copied file. You'll be promted to rename if necessary.

**Reply by omegahh** [reply]
Thanks for replying. I have tried many times, but it does not work.

**Reply by alex-courtis** [reply]
That's not good...

Please submit a [bug report](https://github.com/nvim-tree/nvim-tree.lua/issues/new?assignees=&labels=bug&projects=&template=bug_report.yml) with your configuration and steps to reproduce.

**Reply by omegahh** [reply]
It seems like something wrong in my config file. Now it works very fine. Thank you for doing this!
